app/helpers/application_helper.rb

```diff
@@ -27,6 +27,14 @@ def strip_url(url)
     url.to_s.chomp('/').reverse.chomp('/').reverse.strip.downcase
   end
 
+  def detached_edit_link(flow)
+    if flow[:type] == 'flow.branch'
+      edit_branch_path(service.service_id, flow[:uuid])
+    else
+      edit_page_path(service.service_id, flow[:uuid])
+    end
```

Comment:
If you are using `exclude_branches: true` below, why do we need this if?

Reply:
we only use `exclude_branches: true` when creating the destination list on the edit branch page.

on the service flow page it is possible for a branch to be detached so we need to be able to cater for editing both a page and a branch.